app/helpers/tournaments_helper.rb

```diff
@@ -1,2 +1,34 @@
 module TournamentsHelper
+
+    def success_per_question(results)
+        statistics = Array.new;
+        questions = Array.new;
+    
+        # initialize empty array
+        for index in 0..9 do
+          questions[index] = 0;
+        end
```

Comment:
```suggestion
        questions = Array.new(10, 0);
```

does the same thing :)